app/controllers/api/v1/accounts/assignment_policies_controller.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+class Api::V1::Accounts::AssignmentPoliciesController < Api::V1::Accounts::BaseController
+  before_action :fetch_assignment_policy, only: [:show, :update, :destroy]
+  before_action :check_authorization
+
+  def index
+    @assignment_policies = Current.account.assignment_policies.includes(:inboxes)
```

Comment:
why do we need inboxes here? Please create `policy/:id/inboxes` as a separate endpoint.